docs/pages/config.mdx

```diff
@@ -35,7 +82,7 @@ export default defineWorld({
         },
       },
       systems: {
-        TestSystem: {
+        RootSystem: {
```

Comment:
can we think of another example? I don't think should promote "root" things

Reply:
I changed it. But in general, you often have to have some code with admin powers, and that code needs to be of limited availability.